client/blocks/comment-detail/index.jsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CommentDetailComment from './comment-detail-comment';
+import CommentDetailHeader from './comment-detail-header';
+import CommentDetailPost from './comment-detail-post';
+import CommentDetailReply from './comment-detail-reply';
+
+export class CommentDetail extends Component {
+	static propTypes = {
+		author: PropTypes.object,
+		comment: PropTypes.object,
+		post: PropTypes.object,
+	};
+
+	state = {
+		isExpanded: false,
+	};
+
+	toggleExpanded = () => {
+		this.setState( { isExpanded: ! this.state.isExpanded } );
+	}
+
+	render() {
+		const {
+			comment,
+			post,
+		} = this.props;
+		const { isExpanded } = this.state;
+
+		const classes = classNames( 'comment-detail', {
+			'is-expanded': isExpanded,
+		} );
+
+		return (
+			<Card is-compact className={ classes }>
+				<CommentDetailHeader
+					author={ comment.author }
+					commentBody={ comment.body }
+					isExpanded={ isExpanded }
+					toggleExpanded={ this.toggleExpanded }
+				/>
+				{ isExpanded &&
```

Comment:
Was [FoldableCard](https://wpcalypso.wordpress.com/devdocs/design/foldable-card) not a good fit?